src/platform/user/profile/utilities/index.js

```diff
@@ -195,6 +196,15 @@ export const getValidationMessageKey = (
 };
 
 export const showAddressValidationModal = suggestedAddresses => {
+  if (
+    suggestedAddresses.length === 1 &&
+    suggestedAddresses[0]?.addressMetaData?.confidenceScore > 80 &&
+    suggestedAddresses[0]?.addressMetaData?.deliveryPointValidation ===
+      CONFIRMED
+  ) {
```

Comment:
I noticed that throughout this function `suggestedAddresses[0]?.addressMetaData` is accessed 5 times (6 if my other comment is right).  It may be beneficial to destructure that out into a const so you could have something like:


```js

const [ { addressMetaData } = {} ] = suggestedAddresses;

if (
 suggestedAddresses.length === 1 &&
 addressMetaData?.confidenceScore > 80 &&
 addressMetaData?.deliveryPointValidation ===
    CONFIRMED
) {
```